pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Transactions.java

```diff
@@ -34,6 +35,21 @@
 
 public interface Transactions {
 
+    /**
+     * List transaction coordinators.
+     *
+     * @return the transaction coordinators list.
+     */
+    Map<Integer, TransactionCoordinatorInfo> listTransactionCoordinators() throws PulsarAdminException;
```

Comment:
question (not a request for a change):
why returning a Map and not a List ?
I don't think that returning a Map has a particular advantage over returning a List

Reply:
moved to List